tpm2/constants.go

```diff
@@ -228,3 +230,10 @@ func digestSize(alg Algorithm) int {
 }
 
 const defaultRSAExponent = 1<<16 + 1
+
+var toGoHashes = map[Algorithm]func() hash.Hash{
```

Comment:
I find the name a little ambiguous, since it says what the hash maps to, but not what it maps from. I'm not sure that it needs to say that it's converting to the Go programming language, given that it's written in Go.  How about `hashConstructors`?

Reply:
Done